wow-apps/symfony-slack-bot/3.2/config/packages/wow_apps_slack.yaml

```diff
@@ -0,0 +1,12 @@
+# SlackBot Configuration
+# Documentation https://github.com/wow-apps/symfony-slack-bot/wiki
+wow_apps_slack:
+    api_url: ''
+    default_icon: 'http://cdn.wow-apps.pro/slackbot/slack-bot-icon-48.png'
```

Comment:
http, not https ? this looks bad to me (slack.com forces HTTPS)

Reply:
this is url for default icon, which hosted on my site. Developers can set here custom icon

Reply:
A network url might be more secured, like `//cdn.wow-apps.pro/slackbot/slack-bot-icon-48.png`, as your website allows both http & https